portfolio/src/main/webapp/script.js

```diff
@@ -63,18 +63,32 @@ function getCommentSectionFromServer() {
 function createIndividualComment(comment) {
   const liElement = document.createElement('li');
   // This container is used for showing the username.
-  const usernameElement = document.createElement('p');
+  const usernameElement = document.createElement('div');
+  usernameElement.setAttribute('class', 'username-container');
   // This container is used for showing the text of the comment.
-  const commentElement = document.createElement('p');
+  const commentElement = document.createElement('div');
+  commentElement.setAttribute('class', 'comment-text');
+  // Made an image represent the delete button.
+  const deleteButtonElement = document.createElement('img');
+  deleteButtonElement.setAttribute('class', 'delete-button');
+  deleteButtonElement.src = 'images/trash.png';
+  deleteButtonElement.addEventListener('click', () => {
+    console.log('here');
```

Comment:
I think these were meant as debug logs and you intended to remove them before sending as a PR? :)

Reply:
yes, oops